nix/haskell.nix

```diff
@@ -80,9 +89,6 @@ let
       # TODO: Compile all local packages with -Werror:
       { packages.cardano-tx-generator.configureFlags = [
           "--ghc-option=-Wall"
-          "--ghc-option=-fprof-auto-top"
```

Comment:
did we run it with profiling by default?

Reply:
There was instrumentation, but it wasn't being activated, so I'd say yes and no.

Reply:
The GHC documentation says (https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/profiling.html#compiler-options-for-profiling):
```
Without a -prof option, your SCCs are ignored; so you can compile SCC-laden code without changing it.
```

..so that means that since we don't pass `-prof` by default, these flags should have no effect.